docs/How-to-release.md

```diff
@@ -43,7 +43,7 @@ cd skywalking/cli && svn add "$VERSION" && svn commit -m "Draft Apache SkyWalkin
 
 ## Make the internal announcement
 
-Send an announcement email to dev@ mailing list.
+Send an announcement email to dev@ mailing list, **please check all links before sending the email**, the same below.
```

Comment:
I am not sure about the new words

Reply:
What are you referring to? `the same below` ?

What I mean is that you have to check the availability of each link in the email before you send it out.

Reply:
For myself, it is rare to see the term `the same below` in the documentation. Anyway, just my feeling.

Reply:
You are right, do you have any advice, I didn't find a suitable expression. Otherwise I will delete it.

Reply:
@fgksgf I prefer to simply copy this `**please check all links before sending the email**` to every place when need to send an email, because there are 48+/72+ hours between these emails are sent, release managers are tend to forget and won't look back to the previous steps

Reply:
It makes sense.

Reply:
Make sense to me.